src/mappings.go

```diff
@@ -176,6 +139,19 @@ func convertMappingsJsonToMappings(json MappingsJson) (Mappings, error) {
 	return m, nil
 }
 
+func mergeMappingsFromDefault(dist *Mappings, platform string) error {
```

Comment:
You need not to use pointer for `map` type. In golang, map is treated with reference. Hence we can simply use `m map[K]V` for parameter. Here please simply remove `*`.